### PR TITLE
Default float to decimal

### PR DESCRIPTION
**Pull request by agitator**

### CODE REVIEW

**Comment by jensens** [comment]
Does the upgrade step delete the current settings?

**Comment by agitator** [comment]
@jensens yes, imho the only way to change the field/value type

File: src/bda/plone/shop/upgrades/configure.zcml [hunk 1]
```diff
@@ -92,10 +92,22 @@
...
 
   </genericsetup:upgradeSteps>
 
+  <genericsetup:upgradeSteps
+    source="9"
+    destination="10"
+    profile="bda.plone.shop:default">
+
+    <genericsetup:upgradeDepends
+      title="GS Profile: Update shipping and item defaults from float to decimal"
```
Comment (jensens) [inline src/bda/plone/shop/upgrades/configure.zcml:107]: Add a warning here: current defaults are reset.

Comment (jensens) [reply src/bda/plone/shop/upgrades/configure.zcml:107]: And maybe also in the change log

**Pull request merged**